testing/rstcheck/APKBUILD

```diff
@@ -0,0 +1,32 @@
+# Contributor: Keith Maxwell <keith.maxwell@gmail.com>
+# Maintainer: Keith Maxwell <keith.maxwell@gmail.com>
+pkgname=rstcheck
+_pyname=rstcheck
```

Comment:
```suggestion

```

Reply:
This variable is in line with the draft policy at <https://wiki.alpinelinux.org/wiki/Python_package_policies>. It was deliberately added to facilitate cross referencing against <https://pypi.org>. Perhaps if it should be removed the linked policy on the wiki could be clarified first.